schemas/usage.json

```diff
@@ -711,6 +733,172 @@
 						}
 					},
 					"additionalProperties": false
+				},
+				"settings": {
```

Comment:
Please (and always) attach also an example of the data on top of the tree.

Reply:
Handled.